hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -485,10 +485,19 @@
     <description>
       Comma-separated list of OM service Ids.
 
-      If not set, the default value of "om-service-value" is assigned as the
+      If not set, the default value of "omServiceIdDefault" is assigned as the
       OM service ID.
     </description>
   </property>
+  <property>
+    <name>ozone.om.internal.service.id</name>
```

Comment:
Can we please use the Java Configs instead of this old style config to add a config ?

Reply:
I will open a new Jira to do this, as few of the configs of HA are in old style and few will be in new style.(If we modify for this property)
For now, I will leave this as it is.

Reply:
@bharatviswa504 If some of the HA configs are already in new style, then it makes sense to modify this property here. 
I think @anuengineer would have rather preferred the new style as this is a new property that we are adding, so there is not much gain by using the old style now and then in few days we change it to new style.

Reply:
yes, all ha config is in old-style, so as this is one config change, I have done this way. As said, I will open a new Jira to make it to new style for entire HA config, as it will also help during reading code (All configs can be looked at a single place)

Reply:
https://issues.apache.org/jira/browse/HDDS-2581

Reply:
Ok, SGTM.